### PR TITLE
[skip-ci] Add CMakeLists.txt in test/RootShower

### PR DESCRIPTION
**Pull request by bellenot**

### CODE REVIEW

File: test/RootShower/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+
+project(RootShower)
+
+find_package(ROOT REQUIRED)
+include(${ROOT_USE_FILE})
+
+include_directories(${ROOT_INCLUDE_DIRS} ${CMAKE_SOURCE_DIR})
+
+set(HEADERS
```
Comment (Axel-Naumann) [inline test/RootShower/CMakeLists.txt:16]: This variable seems unused?

**Review by Axel-Naumann — APPROVED** [review]
And see comments, please :-)

File: test/RootShower/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+#
+# For the licensing terms see $ROOTSYS/LICENSE.
+# For the list of contributors see $ROOTSYS/README/CREDITS.
+
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+
+project(RootShower)
+
+find_package(ROOT REQUIRED)
+include(${ROOT_USE_FILE})
```
[the remark this answers is not in the document]
Comment (bellenot) [reply test/RootShower/CMakeLists.txt]: Apparently not... Removed

File: test/RootShower/CMakeLists.txt (hunk 1 above)
Comment (bellenot) [reply test/RootShower/CMakeLists.txt]: Fixed

**Pull request merged**